### ISSUE TITLE
Mythic/Legendary Components arent worth it

### ISSUE THREAD

**Issue by blusave** [onikamigames/starshipgame#454]
Mythic/Legendary Components should perform and feel mythic and legendary. My Legendary ship with Mythic and Legend components are netting LESS Kyanite than my epic ships with Epic and Rare components. If you want to generate excitement and sell more packs next phase, Mythic and Legendary components must be fixed so that they actually benefit the player enough to feel worth it. Otherwise people are just going to buy and use the uncommon available.

**Comment by maaachoooo** [comment]
I agree we spent thousands of dollars just to make 5-10$ on a trip

**Comment by davinseed** [comment]
I have a mythic ship with 2 mythic components. I agree with you. the gdd updated value isn't being fair with us with high rarity ship/nft

**Comment by Double0Sebaz** [comment]
I completely agree and at this point wish I would not have invested as much as I have for mythic ship and parts and a bunch of legendary's... It seems we are worried about "whales" but not taking care of those that have really invested in the game in order to appease the masses.

**Comment by Double0Sebaz** [comment]
I actually compared my Legendary ship with 4 legendary components and the rest epic with a Discord Mod that has an Epic ship with some a epic ship and similar components and his ship had a 1k larger storage capacity than mine!

**Comment by Rowstar** [comment]
My Legendary Ship Is faster then my epic but cost more in fuel to get to planets. They have the same Components but when i do the calculations they both have about the same efficiency in collecting Kyanite.
Is anybody else having this issue?

**Comment by shootypewpew** [comment]
SAME! My god! My Epic ship with all epic parts has strong wfficiency out into the 7th ring but my Legendary ship with a MYTHIC AI and 4 other Legendaries diminishes after 5?!? Devs please, PLEASE make this rebalancing a priority! (And thank you for your hard work thus far)

**Comment by ssecret12345** [comment]
My uncommon ship is now trash compared to my common ones lol. This could be exploited or people could be unaware that they are taking advantage of it.

**Comment by YasuFuma** [comment]
there is a bug right now on re equipping  components, instead of ship being upgraded it rather go the other way, it will get nerfed  now my rare components ship stat are trash compared to my uncommon, stats are like the stat of common XD

**Comment by blusave** [comment]
My original point was that Mythic and Legendary components should have an impact on the game worthy of a mythic and legendary rarity
